### ISSUE TITLE
crash when streaming came (build from latest version)

### ISSUE THREAD

**Issue by xelven** [EricssonResearch/openwebrtc#540]
I used the latest version both of cerbero n openwebrtc build by my self.
```
build version:
cerbero: 4c5700f5c99cd469899c6639ca5e0a315d3faec0
openwebrtc: 30ca989ff4237ebe17641e15fa9cb17c98bf11f9
```
but when finish handshakes and stream came, I got some error from ORC lib and app will crash.
attached below:
![screen shot 2015-12-23 at 11 19 14 am](https://cloud.githubusercontent.com/assets/11630321/11972956/73b3ddd6-a98e-11e5-8c88-c5fa4991b161.png)

seems is gstreamer call that got error, 
did any one had same issue? or some debug mode enable then I can tracking?

thanks,

**Comment by superdump** [comment]
Is it consistently reproducible? Can you click in the bottom-right pane at the lldb prompt and run `bt all` then copy and paste the output to a gist and link it here?

**Comment by xelven** [comment]
@superdump yes, every time I will got this,
here is I paste all of the logs

https://gist.github.com/xelven/440c74f12f379be02106

```bt all``` at the after the ** Message: Got remote source: Remote audio stream

**Comment by superdump** [comment]
The `bt all` is a backtrace of all threads. The reason I asked for it was to see where it had crashed. If you run until it crashes and then run `bt all` and paste that into a gist and link it here, that will be useful for our debugging.

Most people are on holiday currently, so I don't know when someone will get chance to look at it properly.

**Comment by xelven** [comment]
@superdump yes,I did that, I mean I paste ```bt all``` log in that link after line 550.
oh, yeah, Merry Christmas~~

**Comment by xelven** [comment]
updated latest state, 
I roll back checkout cerbero to 
```
2015/12/4
openssl: Bump to 1.0.2e
git:4e8ec2994bb3298db37c3f84356260e86ea3e07a
``` 
then clean n re-build everything is fine.

**Comment by adrianbg** [comment]
Has anyone had a chance to look at this? I'm also seeing a crash in orc_executor_emulate when running NativeDemo from Git with these Pod versions:

  - InAppSettingsKit (2.4.4)
  - OpenWebRTC (0.3.93)
  - OpenWebRTC-SDK (0.3.3):

**Comment by xelven** [comment]
@adrianbg I suggest you roll back the version of cerbero that what I posted in latest.
because this issue relate about integrate between GStreamer n Openwebrtc or some other part of lib.
that won't be easy to fixed.
